README.md

```diff
@@ -30,3 +30,19 @@ docker-compose up
 ```
 
 and nwserver will spin up.
+
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply 
+adding this folder to your solution directory. From there you will need to add this snippet to your `.csproj` file.
+```
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Exec Command="$(SolutionDir)Builder\deployBuild.exe -b $(ProjectDir)$(OutDir)" />
+  </Target>
+``` 
+After that runs your files will be added to `dotnet` folder. From there you `cd` to the directory that has `docker-compose.yml``
```

Comment:
It's probably more accurate to say "Any time your project builds, the binary files will be automatically copied to your dotnet folder" or something to that effect.